BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt

```diff
@@ -354,6 +354,14 @@ open class BraintreeClient @VisibleForTesting internal constructor(
         return browserSwitchClient.getResultFromCache(context)
     }
 
+    /**
+     * @suppress
+     */
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    fun getVersionSDK(): String {
```

Comment:
Instead of adding a getter to the BraintreeClient, would it be cleaner to add the build config field to the PayPal messaging module? 

example: https://github.com/braintree/braintree_android/blob/6b5d8c62977c7c9db5404488d37cc0d31b940f28/GooglePay/build.gradle#L16

Reply:
Thanks!! updated here: [ab41ceb](https://github.com/braintree/braintree_android/pull/981/commits/ab41ceba0cc7d61fb87600567da9b1f3d0614e91)